.gitignore

```diff
@@ -1 +1,5 @@
 *.pyc
+.tox/
+build/
+dist/
```

Comment:
They popped up after I installed the project locally, `python setup.py
install`
On 21 Jun 2016 9:47 p.m., "Romain Garrigues" <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/iwoca/django-deep-collector/pull/25#discussion_r67948562>
> :
>
> > @@ -1 +1,5 @@
> >  *.pyc
> > +.tox/
> > +build/
> > +dist/
>
> Where are build and dist coming from?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/iwoca/django-deep-collector/pull/25/files/83d1d0eaf78c7a63b87a671d5aee1ee7a79c2ba6#r67948562>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABIxehwE9C6Faue3ZlckpYHANywqW_TOks5qOE3zgaJpZM4I7I1d>
> .
>